itests/eth_fee_history_test.go

```diff
@@ -22,13 +24,16 @@ func TestEthFeeHistory(t *testing.T) {
 
 	blockTime := 100 * time.Millisecond
 	client, _, ens := kit.EnsembleMinimal(t, kit.MockProofs(), kit.ThroughRPC())
-	ens.InterconnectAll().BeginMining(blockTime)
+	miner := ens.InterconnectAll().BeginMining(blockTime)
 
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 	defer cancel()
 
 	// Wait for the network to create 20 blocks
 	client.WaitTillChain(ctx, kit.HeightAtLeast(20))
```

Comment:
What happens if some of these rounds are null blocks? I can't tell whether the test assertions are still correct.